src/test/ctl_cow/err3.log.match

```diff
@@ -1 +1 @@
-{ctl_cow.c:145 test_dax} ctl_cow/TEST3: Error: cannot open $(nW)/dax.set: Operation not supported
+{ctl_cow.c:116 test_dax} ctl_cow/TEST3: Error: cannot open $(nW)/dax.set: Operation not supported
```

Comment:
Please replace this with $(N) or even replace everything up to an error message with $(*)

Reply:
Done.